app/services/draft_document_collection_builder.rb

```diff
@@ -56,6 +56,8 @@ def initialise_document_collection
   def add_groups_to_document_collection(document_collection)
     groups =
       specialist_topic_groups.map do |group|
+        next if group[:name].empty?
```

Comment:
This works as longs as it isn't possible for a group to have an array of content id's with no name... do we know that's the case?

Reply:
Hmm that's a good point. All the groups with empty names I saw so far had no content ids. I tried to test it on integration but I can't create a group with empty name so I assume this is an old issue that was fixed.

Reply:
Maybe I can add a placeholder group name for this one as we do with the summary and then the CD can delete the group if it's empty

Reply:
Maybe we should instead do `next if empty_group?(group)` that checks if both heading and content_id's are empty. Just in case. It will blow up again if nameless groups _do_ exist, and we can look into it then.

Reply:
Yes, this is much better. Thanks